test/index/types/types.snapshot.cc

```diff
@@ -43,10 +45,12 @@
   namespace a {
 //          ^ definition [..] a/
```

Comment:
Unrelated, but it would be nice to have multiple `namespace a` in a file ... that just produces multiple definitions?

Reply:
This is covered by the `namespaces.cc` test; yes, it creates multiple definitions.